store/index.js

```diff
@@ -2,7 +2,8 @@ import { createStore, applyMiddleware, combineReducers } from "redux";
 import thunk from "redux-thunk";
 
 import { authReducer as auth } from "./auth/authReducer";
+import { recipeReducer as allRecipes } from "./auth/recipeReducer";
```

Comment:
LIttle mistake I noticed when testing the login in android studio emulator, should be `from  "./recipes/recipeReducer"`